packages/functions/src/ui/adapters/password.ts

```
@@ -308,13 +308,13 @@ export function PasswordAdapter(config: PasswordConfig) {
         return transition({ type: "start", redirect: adapter.redirect })
       })
     },
-  } satisfies Provider<{ email: string; username?:string }>
+  } satisfies Provider<{ email: string; username?: string }>
 }
 
 import * as jose from "jose"
 import { TextEncoder } from "node:util"
 
-interface HashedPassword {}
+interface HashedPassword { }
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)